cmd/crane/cmd/rebase.go

```diff
@@ -179,9 +180,12 @@ func rebaseImage(orig v1.Image, oldBase, newBase string, opt ...crane.Option) (v
 		return nil, fmt.Errorf("either old base or %q annotation is required", specsv1.AnnotationBaseImageDigest)
```

Comment:
`oldBase == ""` will still return an error for backwards compatibility. oldBase needs to be set explicitly to 'scratch' for the new behavior to kick in. Even though this is inconsistent with `crane append --base`